src/assets/scss/components/social-icons.scss

```diff
@@ -10,7 +10,6 @@
         padding: 0;
         margin-left: -1rem;
         margin-inline-start: -1rem;
```

Comment:
if we do the following changes we don't need to wrap the mastodon icon , it appears in the same line even at different breakpoints.
<details>
<summary>Screenshot (1052 x 522)</summary>


![image](https://user-images.githubusercontent.com/42532987/207842062-19c1f2ac-5727-4db4-a856-d9253aeb9131.png)


</details>

```suggestion
        display: flex;
        justify-content: space-between;
```


Remove `padding` from here 

https://github.com/eslint/eslint.org/blob/885d0d939ebc77370f56db475c750893552335e4/src/assets/scss/components/social-icons.scss#L22

Reply:
How does that work on the homepage footer?

Reply:
we need to change the properties little bit to achieve that.

Reply:
Added nowrap changes with socials layout fix in https://github.com/eslint/eslint.org/pull/386 (homepage and playground)